lib/elixir/src/elixir.erl

```diff
@@ -297,3 +297,11 @@ string_to_quoted(String, StartLine, File, Opts) when is_integer(StartLine), is_b
 
 to_binary(List) when is_list(List) -> elixir_utils:characters_to_binary(List);
 to_binary(Atom) when is_atom(Atom) -> atom_to_binary(Atom, utf8).
+
+handle_parsing_opts(File, Opts) ->
+  put(elixir_parser_file, File),
+  case lists:keyfind(wrap_literals_in_blocks, 1, Opts) of {wrap_literals_in_blocks, WrapLiteralsBool} when is_boolean(WrapLiteralsBool) ->
```

Comment:
I like your way better. Can I replace `Bool` with `true` instead? Since `Bool` can be non-boolean. Ideally, guard should be used to capture booleans but I think it gets too verbose to be readable.

```erlang
WrapLiteralsInBlocks =
  case lists:keyfind(wrap_literals_in_blocks, 1, Opts) of
    {wrap_literals_in_blocks, true} -> true;
    _ -> false
  end,
```